main/Kconfig.projbuild

```diff
@@ -68,11 +68,7 @@ menu "Monster Configuration"
             Minimum CPU frequency to use for dynamic frequency scaling.
             Should be set to XTAL frequency or XTAL frequency divided by integer.
 
-        config EXAMPLE_MIN_CPU_FREQ_40M
-            bool "40 MHz (use with 40MHz XTAL)"
-            depends on IDF_TARGET_ESP32S2 || ESP32_XTAL_FREQ_40 || ESP32_XTAL_FREQ_AUTO
-        config EXAMPLE_MIN_CPU_FREQ_20M
-            bool "20 MHz (use with 40MHz XTAL)"
+        config EXAMPLE_MIN_CPU_FREQ_40M  #endif)"
```

Comment:
Is this change intentional? I think this is likely incorrect

Reply:
Yeah I have no clue why that changed, I did not change it